eng/packaging.targets

```
@@ -57,6 +58,11 @@
                                   TargetFramework="$(NetFrameworkMinimum)" />
   </ItemGroup>
 
+  <!-- Add a package README file from. -->
+  <ItemGroup Condition="'$(PackageReadmeFile)' != ''">
+    <None Include="$(PackageReadmeFile)" Pack="true" PackagePath="\" />
```

```suggestion
    <None Include="$(PackageReadmeFile)" Pack="true" PackagePath="README.md" />
```
You want the file to be named `README.md`, right?

---

We don't have to specify name README.md, NuGet will recognize any name set by PackageReadmeFile property. Also PackagePath seems to be a path without name, it is used like that in docs: https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#packagereadmefile